src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,58 @@
+package duke;
+
+import duke.command.Command;
+
+import java.util.Scanner;
+
+/**
+ * Duke is a personal assistant that keep a list of your tasks. Users
+ * are able to add different types of tasks to their own list as well as
+ * mark tasks as completed or delete tasks from their task list.
+ *
+ * @author leezhixuan
+ */
+
+public class Duke {
+
+    private static String name = "Duke";
+    private boolean isRunning = false;
+    private ToDoList tdl;
```

Comment:
Maybe for the tdl variable, you could type change to toDoList? so it would be easier to read your code later on 👍